src/generated-client.js

```diff
@@ -5355,7 +5355,7 @@ class GeneratedApiClient {
    * @returns {Promise<User>}
    */
   expirePassword(userId) {
-    let url = `${this.baseUrl}/api/v1/users/${userId}/lifecycle/expire_password?tempPassword&#x3D;false`;
+    let url = `${this.baseUrl}/api/v1/users/${userId}/lifecycle/expire_password?tempPassword=false`;
```

Comment:
Was this code generated by the modifications to the template?

Reply:
Yes. The whole file is generated.